### ISSUE TITLE
rosjava services cannot be called(indigo)

### ISSUE THREAD

**Issue by bbrieber** [rosjava/rosjava_core#202]
If I try to call a rosjava service in indigo, the call fails with the message:

```
 ERROR: unable to connect to service: 'callerid 
```

This happens for all services(they are working in hydro)

I followed the installation instructions from the ROS-wiki (http://wiki.ros.org/rosjava/Tutorials/indigo/Installation)

The bug can be reproduced on different PCs by starting the Demo server

```
roscd rosjava_core
cd rosjava_tutorial_services/build/install/rosjava_tutorial_services/bin
./rosjava_tutorial_services org.ros.rosjava_tutorial_services.Server
```

and then calling the service from command line 

```
rosservice call /add_two_ints "a: 0 b: 2" 
```

**Comment by Sanic** [comment]
As we found out today, this error seems to be fixed if you now update all the packages provided by the rosjava installation.
Assuming that you installed rosjava from source as stated here: http://wiki.ros.org/rosjava/Tutorials/indigo/Installation
- cd into the Workspace where rosjava is
- cd src
- wstool update
- cd ..
- catkin_make

I didn't investigate this further, but it might be that the rosservice command has been fixed. Since roscpp is delivered with rosjava and it overlays the normal workspaces, you will use the rosservice command from the code out of the rosjava stuff.
